extension/src/background/messageListener/popupMessageListener.ts

```diff
@@ -84,21 +93,21 @@ export const popupMessageListener = (request: Request) => {
       console.error(e);
     }
 
+    const keyIdListStr = localStorage.getItem(KEY_ID_LIST) || "[]";
+    const keyIdListArr = JSON.parse(keyIdListStr);
+    keyIdListArr.push(keyStore.id);
+
+    localStorage.setItem(KEY_ID_LIST, JSON.stringify(keyIdListArr));
```

Comment:
we're storing an array of keyId's in localStorage. We will keep a concept of an "active" keyId, meaning the one the user is currently working with